tests/apps/courses/test_models_blogpost.py

```diff
@@ -22,3 +23,57 @@ def test_models_blogpost_str(self):
         blogpost = BlogPost(extended_object=page)
         with self.assertNumQueries(1):
             self.assertEqual(str(blogpost), "Blog Post: My first article")
+
+    @staticmethod
+    def _attach_category(blogpost, category):
+        """Not a test. Utility method to easily attach a category to a blogpost."""
+        placeholder = blogpost.extended_object.placeholders.get(slot="categories")
+        add_plugin(placeholder, "CategoryPlugin", "en", page=category.extended_object)
+
+    def test_models_blogpost_get_related_blogposts_plugin_on_draft(self):
```

Comment:
It's not `on_draft` since you're also testing public blogposts?

Reply:
Good catch. I intially had 2 tests ;-)